examples/comparison_ccl_nc_theory_backend.ipynb

```diff
@@ -0,0 +1,232 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#del os.environ['CLMM_MODELING_BACKEND']\n",
+    "\n",
+    "#os.environ['CLMM_MODELING_BACKEND'] = 'ccl' \n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "14677ba5-4090-496a-a0f5-43fbb280246a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cosmo = Cosmology(H0=70.0, Omega_dm0=0.27-0.045, Omega_b0=0.045, Omega_k0=0.0)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "2a477f91-5421-4bba-8067-94d627434d19",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "moo = clmm.Modeling(massdef='mean', delta_mdef=200, halo_profile_model='nfw')\n",
+    "\n",
+    "moo.set_cosmo(cosmo)\n",
+    "moo.set_concentration(4)\n",
+    "moo.set_mass(1.e15)\n",
+    "\n",
+    "z_cl = 1.\n",
+    "\n",
+    "#source properties\n",
+    "z_source = 2. #all sources in the same plane\n",
+    "z_distrib_func = clmm.utils._chang_z_distrib #sources redshift following a distribution\n",
+    "alpha = [2, -0.5] "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "55092bae-31ad-4d3d-aec7-fc68d1417744",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "r3d = np.logspace(2, 2, 100)\n",
+    "rho = moo.eval_3d_density(r3d, z_cl)\n",
+    "Sigma = moo.eval_surface_density(r3d, z_cl)\n",
+    "DeltaSigma = moo.eval_excess_surface_density(r3d, z_cl)\n",
+    "Sigmac = moo.eval_critical_surface_density(z_cl, z_source)\n",
+    "gammat = moo.eval_tangential_shear(r3d, z_cl, z_source)\n",
+    "kappa = moo.eval_convergence(r3d, z_cl, z_source)\n",
+    "\n",
+    "gt_ccl = moo.eval_reduced_tangential_shear(r3d, z_cl, z_source)\n",
+    "#Lensing quantities assuming sources follow a given redshift distribution.\n",
+    "gt_z_ccl = moo.eval_reduced_tangential_shear(r3d, z_cl, z_source)\n",
+    "\n",
+    "mu_ccl = moo.eval_magnification(r3d, z_cl, z_source)\n",
+    "mu_bias_ccl = moo.eval_magnification_bias(r3d, z_cl, z_source, alpha)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "08ef22cf-de6e-44ea-bcca-198e7b30c04f",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "1.000014511204969 7.253514796379729e-06 6.338310252511474e-05 6.338356227872158e-05\n"
+     ]
+    }
+   ],
+   "source": [
+    "#CCL results\n",
+    "print(mu_ccl[0], kappa[0], gammat[0], gt_ccl[0])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "39841410-cef3-4d86-8c3b-189b9044817a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "1.0000145112041072 7.253514366217001e-06 6.338309030485296e-05 6.338355005834388e-05\n"
+     ]
+    }
+   ],
+   "source": [
+    "#NumCosmo results\n",
+    "print(1.0000145112041072 ,7.253514366217001e-06 ,6.338309030485296e-05, 6.338355005834388e-05)"
```

Comment:
The results seem to mach up to 5 decimals, but in the 6th decimal, ccl and numcosmo are different for some values.